src/tests/test_cases/test.py

```diff
@@ -316,8 +316,13 @@ def sdgfsdg23245777():
 
 
 @expect('D402: First line should not be the function\'s "signature"')
-def foobar():
-    """Signature: foobar()."""
+def foobar(foo=1, bar=1):
```

Comment:
These look good. Can you also add some tests which are false negatives to ensure we *don't* raise errors for them?

Reply:
DONE